Rover/AP_Rally.h

```diff
@@ -17,8 +17,7 @@
 #include <AP_Rally/AP_Rally.h>
 #include <AP_AHRS/AP_AHRS.h>
 
-class AP_Rally_Rover : public AP_Rally
-{
+class AP_Rally_Rover : public AP_Rally {
```

Comment:
@peterbarker , @khancyr ,

Please confirm, If you want me to revert braces related changes in .h files as well.